lib/bolt/outputter/human.rb

```diff
@@ -78,7 +78,13 @@ def print_summary(results, elapsed_time)
       end
 
       def print_plan(result)
-        @stream.puts result
+        if result_hash = unwrap_execution_result(result)
+          unless result_hash.empty?
```

Comment:
Chose to avoid printing anything - rather than `[\n\n]\n` - for human format.

Reply:
I think there needs to be something here  `[ ]` seems better than that though

Reply:
That implies an empty result?

Reply:
Isn't it an empty result?